packages/create-cloudflare/src/cli.ts

```diff
@@ -41,6 +42,13 @@ import type { C3Args, C3Context } from "types";
 const { npm } = detectPackageManager();
 
 export const main = async (argv: string[]) => {
+	if (process.versions.bun || npm === "bun") {
+		console.warn(
```

Comment:
There was a regression in Bun v1.2.0 that impacted this, but that regression was fixed in Bun v1.2.1. I manually verified it works without error in Bun v1.2.1 and Bun v1.2.2 (and that it regressed in Bun v1.2.0, which matches what you saw in the CI runs). We are adding more tests from Node.js' test suite to catch bugs like this in the future.

Is this warning still needed? We will fix anything else that comes up

Reply:
I think so. Bun’s continued disregard for ecosystem compatibility puts burden on third party maintainers, as well as the community where often “use Node.js” is the solution to most issues once Bun is discovered to be used.

In my opinion, until Bun commits to join something like WinterTC and work closely with other runtimes before implementing ecosystem breaking or diverging features, I don’t imagine this burden will change.